crates/web5_cli/install.sh

```diff
@@ -28,12 +52,24 @@ case $OS in
 esac
 
 # Download
-curl -L -o /tmp/$FILENAME https://github.com/TBD54566975/web5-rs/releases/download/$VERSION/$FILENAME
+echo "Downloading $FILENAME"
+curl -L -f -o /tmp/$FILENAME https://github.com/TBD54566975/web5-rs/releases/download/$VERSION/$FILENAME
+
+# Check download errors
+if [ $? -ne 0 ] ; then
+  echo "Error while downloading $FILENAME"
+  echo "Exiting..."
+  exit 1
+fi
 
 # give it executable permissions
 chmod +x /tmp/$FILENAME
 
 # Move the executable to /usr/local/bin
+if [ -d "/usr/local/bin" ]; then
+  echo "Creating /usr/local/bin"
+  mkdir /usr/local/bin
+fi
```

Comment:
```suggestion
if [ ! -d "/usr/local/bin" ]; then
  echo "Creating /usr/local/bin"
  mkdir -p /usr/local/bin
fi
```

not sure what this woudl do on windows